### PR TITLE
Add LRA Listops and IMDB experiments

### PR DESCRIPTION
**Pull request by jxiw**
### ListOps

Downolad dataset in [GitHub page](https://github.com/google-research/long-range-arena). Unzip the downloaded dataset, and move it to the project folder.

```
python -m s4.train --dataset listops-classification --model s4 --epochs 100 --bsz 50 --d_model 128 --n_layers 4 --ssm_n 64 --lr 8e-3 --p_dropout 0
```

Training with the previously defined hyper-parameters yields the test accuracy 50.2% on the test set.

Screen Shot 2022-04-08 at 10 43 04 AM

### IMDB

We used the huggingface datasets for LRA IMDB review task.

```
python -m s4.train --dataset imdb-classification --model s4 --epochs 30 --bsz 50 --d_model 64 --n_layers 4 --ssm_n 64 --lr 1e-2 --p_dropout 0
```

Training with the previously defined hyper-parameters yields the test accuracy 78.42% on the test set.

Screen Shot 2022-04-07 at 2 03 57 AM

### CODE REVIEW

**Comment by srush** [comment]
Need to run the linter. `make autoformat`

**Comment by srush** [comment]
Actually this is not a real error. Turned it off in the linter.
